src/components/MessageInput.js

```diff
@@ -53,6 +53,11 @@ const MessageTextInput = styled.TextInput`
     bottom: 3;
 `;
 
+const MessageText = styled.TextInput`
```

Comment:
I removed MessageText in my last PR, so it is not needed anymore

Reply:
Do I need to rerun the code to see if it still works? or did you remove it after testing?

Reply:
Not sure, you can test to see